src/extension.ts

```diff
@@ -38,7 +38,7 @@ function showDecoration(e: { readonly textEditor: vscode.TextEditor }) {
     cmd.stdin.end();
   }
 
-  const gitUser = cp.execSync("git config user.name").toString().trim();
+  const gitUser = cp.execSync("git config user.name").toString().trim() || "John Doe";
```

Comment:
Hehe, thanks!

Assume `git config user.name` returns an empty string and doesn't crash when unset.

Reply:
yes, tested that.